### DISCUSSION TITLE
Hi - found rAudio-1 a couple of days ago and loaded latest 2205526 image on my RPi 4B 8Gb.

### DISCUSSION THREAD

**Discussion by Lightwel** [rern/rAudio#938] [Tips]
Working well, but I can't see how to get 'under the hood' from the web gui. How do I get to a terminal / file editor?

It must be obvious but I'm missing it. Tried TTY but nothing. Tried opening with screen and kb, no change.

Do I have to build from scratch? (Might try that anyway.) 

Tks

**Comment by R101G** [comment]
Try ssh with password ros.

**Reply by Lightwel** [reply]
Of course .. Tks. Done.

A couple of other questions if I can: 

1 how to transfer files permanently to USB - smb only seems to mount my source directory temporarily: I may be misunderstanding smb. Is there a GUI way of doing this easier than mounting the source and cp using the cli?

2  is there a GUI way of seeing free space? 

Tks again

**Comment by R101G** [comment]
Go to System/Storage to see free space. Click on the + symbol to add the remote storage, then create a library shortcut.. Also, please look through every option on every menu, so that you have some idea of what settings are available and which ones might be useful to you. Update the library manually or reboot when you change anything.

**Reply by Lightwel** [reply]
Ok had a look .. 

Free space on /boot is shown, but not on /mnt/MPD/USB/ROOT (which shows a red dot). The info drop down shows a sector size of 512MB and a device size of 0MB. The /boot /root fs are on a 32Gb USB drive. Should this be showing a different result? 

Perhaps my question was not clear - not how to mount remote storage, but how to transfer files into the existing USB storage. 

There is something in the GIF instructions about drag and drop to player, but it goes past v quickly. Is that relevant? 

Tks
